### DISCUSSION TITLE
how to change the build-in style of editor?  I want to change the gjs-selected style， what should i do？

### DISCUSSION THREAD

**Discussion by pittleCheung** [GrapesJS/grapesjs#5203] [Q&A]
<style>
      
    body { background-color: #fff }
    * ::-webkit-scrollbar-track { background: rgba(0, 0, 0, 0.1) }
    * ::-webkit-scrollbar-thumb { background: rgba(255, 255, 255, 0.2) }
    * ::-webkit-scrollbar { width: 10px }
  

      [data-gjs-type="wrapper"] {
        min-height: 100vh;
        padding-top: 0.001em;
      }

      .gjs-dashed *[data-gjs-highlightable] {
        outline: 1px dashed rgba(170,170,170,0.7);
        outline-offset: -2px;
      }

      .gjs-selected {
        outline: 2px solid #3b97e3 !important;
        outline-offset: -2px;
      }

      .gjs-selected-parent {
        outline: 2px solid #ffca6f !important
      }

      .gjs-no-select {
        user-select: none;
        -webkit-user-select:none;
        -moz-user-select: none;
      }

      .gjs-freezed {
        opacity: 0.5;
        pointer-events: none;
      }

      .gjs-no-pointer {
        pointer-events: none;
      }

      .gjs-plh-image {
        background: #f5f5f5;
        border: none;
        height: 100px;
        width: 100px;
        display: block;
        outline: 3px solid #ffca6f;
        cursor: pointer;
        outline-offset: -2px
      }

      .gjs-grabbing {
        cursor: grabbing;
        cursor: -webkit-grabbing;
      }

      .gjs-is__grabbing {
        overflow-x: hidden;
      }

      .gjs-is__grabbing,
      .gjs-is__grabbing * {
        cursor: grabbing !important;
      }

      
      * { box-sizing: border-box; } body {margin: 0;}
    </style>

**Comment by pittleCheung** [comment]
and how to reset  .gjs-hovered style

**Comment by artf** [comment]
You can use [`canvasCss` option](https://github.com/GrapesJS/grapesjs/blob/f16638a0f3da126dd68af069975b915f0569fb98/src/editor/config/config.ts#L149-L154)
